crates/uv-pep440/src/version_specifier.rs

```diff
@@ -554,16 +554,9 @@ impl VersionSpecifier {
                 other.as_ref() >= this
             }
             Operator::GreaterThan => Self::greater_than(this, &other),
-            Operator::GreaterThanEqual => {
-                Self::greater_than(this, &other) || other.as_ref() >= this
-            }
-            Operator::LessThan => {
-                Self::less_than(this, &other)
-                    && !(version::compare_release(&this.release(), &other.release())
-                        == Ordering::Equal
-                        && other.any_prerelease())
-            }
-            Operator::LessThanEqual => Self::less_than(this, &other) || other.as_ref() <= this,
+            Operator::GreaterThanEqual => other.as_ref() >= this,
+            Operator::LessThan => Self::less_than(this, &other),
```

Comment:
Is there a Reason some of these are Proper operators and others are named methods?

Reply:
Addressed in #12873